soaks/Dockerfile.builder

```diff
@@ -1,8 +1,18 @@
-FROM docker.io/rust:1.58-bullseye@sha256:d83bf5ea7b4c3d18c2f46d5f3d288bfca085c3e7ac57822e3b8e5a1ad22ccc1a as builder
-RUN apt-get update && apt-get -y install build-essential git clang cmake libclang-dev libsasl2-dev libstdc++-10-dev libssl-dev libxxhash-dev zlib1g-dev zlib1g
+FROM docker.io/rust:1.58-bullseye@sha256:e4979d36d5d30838126ea5ef05eb59c4c25ede7f064985e676feb21402d0661b as builder
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get -y install build-essential git clang cmake libclang-dev \
+    libsasl2-dev libstdc++-10-dev libssl-dev libxxhash-dev zlib1g-dev zlib1g
+
+# Build mold, a fast linker
 RUN git clone https://github.com/rui314/mold.git
-RUN cd mold && git checkout v1.0.1 && make -j$(nproc) && make install
+RUN cd mold && git checkout v1.1.1 && make -j$(nproc) && make install
 RUN rm -rf mold
 
+# Build lading, the load generating and measuring tool for vector
+RUN git clone https://github.com/blt/lading.git && cd lading && git checkout ba101daed08a4d90264b01b9c1d77b593c98853a && cargo build --release && cp target/release/lading /usr/local/bin/lading
```

Comment:
This will need to change once https://github.com/blt/lading/pull/148 is merged and a proper tag is cut.